### ISSUE TITLE
Reproduce problem on tf random

### ISSUE THREAD

**Issue by lsylusiyao** [SciSharp/TensorFlow.NET#746]
Related to [PR #745](https://github.com/SciSharp/TensorFlow.NET/pull/745).

Usually we use `tf.set_random_seed(n)` to make the training step reproducable. Also, the seed can be set in functions like `tf.random.normal(1, seed: 1234)`. 

However, no matter which part I tried, the constant seed doesn't work. I tried to debug into the code and found that they all went to [`random_ops.cs`](https://github.com/SciSharp/TensorFlow.NET/blob/master/src/TensorFlowNET.Core/Operations/random_ops.cs) and ran by `tf_with`. Here, the seed1 and seed2 will be set to (0,0) or (DEFAULT_GRAPH_SEED, 0) everytime. Though the seeds aren't what I've set above, they're constants so the process SHOULD be reproduced.  When I tried in UnitTest, the result couldn't be reproduced either in one time (copy the same code twice in one function) or two times (run the test twice).

Sorry I can't figure out why. Please help me.

BTW, the same test has been tested in Python-TF2 and it worked out.

**Comment by Sebastian-roko** [comment]
In #762 this problem should be mostly fixed.
